src/mainmatter.css

```diff
@@ -0,0 +1,596 @@
+@font-face {
+  font-family: 'Core Sans A';
+  font-style: normal;
+  font-weight: 400;
+  src: url(/assets/fonts/CoreSansA45.woff);
+}
+
+@font-face {
+  font-family: 'Core Sans A';
+  font-style: italic;
+  font-weight: 400;
+  src: url(/assets/fonts/CoreSansA45It.woff);
+}
+
+@font-face {
+  font-family: 'Core Sans A';
+  font-style: normal;
+  font-weight: 700;
+  src: url(/assets/fonts/CoreSansA65.woff);
+}
+
+@font-face {
+  font-family: 'Core Sans A';
+  font-style: italic;
+  font-weight: 700;
+  src: url(/assets/fonts/CoreSansA65It.woff);
+}
+
+/* Monospace */
+@font-face {
+  font-family: 'Spline Sans';
+  font-style: normal;
+  font-weight: 400;
+  src: url(/assets/fonts/SplineSansMono-VariableFont_wght.ttf);
+}
+
+@font-face {
+  font-family: 'Spline Sans';
+  font-style: italic;
+  src: url(/assets/fonts/SplineSansMono-Italic-VariableFont_wght.ttf);
+}
+
+:root {
+  --font: 'Core Sans A', sans-serif;
+  --font-mono: 'Spline Sans', monospace;
+
+  --code-background: white;
+  --code-border: transparent;
+  --code-text: #6c696e;
+  --code-comment: #a7a5a8;
+  --code-type: #aa17e6;
+  --code-include: #aa17e6;
+  --code-string: #17ad98;
+  --code-class-name: #dc8a0e;
+  --code-numbers: #d65407;
+  --code-variables: #d8137f;
+  --code-functions: #775dff;
+  --code-literal: #d65407;
+
+  /* Mainmatter-specific */
+  --mainmatter-purple: #6200ee;
+  --mainmatter-blue: #307fee;
+  --mainmatter-teal: #02ceba;
+  --mainmatter-orange: #ffc400;
+
+  --font-size-sm: clamp(1.56rem, 0.81vw + 1.35rem, 2rem);
+  --font-size-base: clamp(1.75rem, 1.36vw + 1.41rem, 2.5rem);
+  --font-size-md: clamp(1.97rem, 2.1vw + 1.44rem, 3.13rem);
+  --font-size-lg: clamp(2.21rem, 3.08vw + 1.45rem, 3.91rem);
+  --font-size-xl: clamp(2.49rem, 4.35vw + 1.4rem, 4.88rem);
+  --font-size-2xl: clamp(2.8rem, 6vw + 1.3rem, 6.1rem);
+  --font-size-3xl: clamp(3.15rem, 8.14vw + 1.12rem, 7.63rem);
+}
+
+/* @link https://utopia.fyi/type/calculator?c=320,24,1.125,1240,32,1.25,5,1,&s=0.75|0.5|0.25,1.5|2|3|4|6,s-l&g=s,l,xl,12 */
+
+:root {
+  --step--1: clamp(1.3331rem, 1.2403rem + 0.4641vw, 1.6rem);
+  --step-0: clamp(1.5rem, 1.3261rem + 0.8696vw, 2rem);
+  --step-1: clamp(1.6875rem, 1.4049rem + 1.413vw, 2.5rem);
+  --step-2: clamp(1.8988rem, 1.4722rem + 2.1326vw, 3.125rem);
+  --step-3: clamp(2.1356rem, 1.5198rem + 3.0793vw, 3.9063rem);
+  --step-4: clamp(2.4025rem, 1.5397rem + 4.3141vw, 4.8831rem);
+  --step-5: clamp(2.7031rem, 1.5203rem + 5.9141vw, 6.1038rem);
+}
+
+/**
+ * A simple theme for reveal.js presentations, similar
+ * to the default theme. The accent color is darkblue.
+ *
+ * This theme is Copyright (C) 2012 Owen Versteeg, https://github.com/StereotypicalApps. It is MIT licensed.
+ * reveal.js is Copyright (C) 2011-2012 Hakim El Hattab, http://hakim.se
+ */
+@import url(https://fonts.googleapis.com/css?family=News+Cycle:400,700);
+@import url(https://fonts.googleapis.com/css?family=Lato:400,700,400italic,700italic);
```

Comment:
These are causing the server to complain but I don't think they are actually required